### ISSUE TITLE
Update and Correction: PLAN 05X series [WIP]

### ISSUE THREAD

**Issue by RickDNiu** [PygmalionOfCyprus/cmo-db-requests#5097]
### List

- https://docs.google.com/spreadsheets/d/1VMFmcDnw6I0PMsxI6W7uDYMNF4AheVOzjzdXX-agWmw/edit?usp=sharing [WIP]

---
### Mounts

- [WIP]

---
### Sensors 

- **MFR**

- H/LJG-346/346 Improved/346A/346B S band MFR, 14th institute [^346]
1. Type 346, 2004, early 1990s tech. (Originally planned for 052B but bidding/protest delayed its selection), 052C Batch 1
S band search. 4768 modules grouped in fours into a 20x5x45cm box, with almost 100W maximum power for each unit. Hence there's 1192 units and 119.2kW max power. Each array were 4 tons. 43.33dBi with -102dBm MDS. 375km range and 320km +-60deg azimuth (range numbers are as original, probably a F-16A front target as baseline, as 2sqm/2GHz test shows the same result).
C band element provides for TVM guidance up/downlink, no tracking/SARH. 2x 0.2x4m 0.8sqm arrays. 8 channel. 32 on one ship.
Air cooled, would be vulnerable in cases when the curved aero cover was to be damaged.

2. Type 346 Improved, late 1900s tech. 052C Batch 2
S band search. 5000+ modules. Miniaturized into 20x5x40cm boxes, lighter structural material, 4 tons. Proposed 130kW, 43.7dBi, -102dBm MDS. 400km range and 375km +-60deg azimuth (range numbers are as original, probably a F-16A front target as baseline, as 2sqm/2GHz test shows the same result).
C band provides for TVM guidance up/downlink, no tracking/SARH. 2x 1sqm, 8 channel. 32 on one ship.
Air cooled. 

3. Type 346A, late 2000s tech. ABM capable. 052D. 
S band search. 5000+ modules. 400km range and 375km +-60deg azimuth. Calibration Target removed on second batch. 
C band antenna removed. HQ-9 guidance handled by SR-2410C. Improved SWAP meant separate radar/antenna were feasible. 
Liquid cooled flat surface.

4. Type 346B, early 2020s tech. NCTR. 055.
S band search. 40% larger and said to have 60% greater range, GaN TR modules. Assuming proportional scale and 2x power delivery: 360kW, 45dBi, -103dBm MDS. ~640km for 2sqm target, which is compatible with the 60% increase.
Liquid cooled flat surface.

- H/LJQ-368 MFR, early 2020s tech, 052D4 (as low alt/surface search), 055 (as X band search), 054A6/B (as MFR)

AESA multi band (S?/C/X) radar, replacing 364 on 052D, 382, 366, 364/Mineral-ME (main search/HQ-16 beacon response, low alt/surface search/FC, surface target indication/OTH) on 054A, X band search on 055. 
1x Dual faced rotating on 052D4/054A6/B, 4x fixed (facing F/R/P/S) on 055 (X band component only?). 

---
- **FCR**

- MR-35/H/LJP-345

FCR for HQ-7

- MR-90

FCR for SA-N-7/17/HQ-16, 052B, 053X, 054A1-5 [WIP]

- SR-2410C C band Track/IFF/FCR, late 2010

AESA tracking/IFF/FCR with similar role to SPY-3/APAR, provides for IFF, HQ-9 TVM on 052D (4x, one above each 364A)/055 (6x, 4 above each 364B and two forward/rear), and HQ-16 TVM + terminal SARH illumination on 054B (4x, facing F/R/P/S, replacing MR-90). Reconfigured rotating as MFR with volume search on 054AP for Pakistan. 
Proposed 8 channel DL. 

- H/LJP-344/344A

Main gun FCR

- H/LJP-349

CIWS FCR

---
- **Ssearch**

- H/LJQ-517

1. H/LJQ-517B

2. H/LJQ-517C

- H/LJQ-366

BVR Air/surface search, surface target indication/ASCM FCR

- H/LJQ-364

Minerial-ME copy

---

- **EW**

- H/RJZ-726

IFF/ESM(VHF/UHF)/RWR/ECM(VHF/UHF); Mast; 1x IFF/ESM/RWR, 2x ECM 

- NRJ-6

1. NRJ-6, 

ESM/ECM/EO/IR/LWR; superstructure; 2x ESM/ECM/EO/IR/LWR 

2. NRJ-6A, early 2010, AESA

ESM/ECM/EO/IR/LWR; superstructure; 2x ESM/ECM/EO/IR/LWR 

---
### Comms

- 

[^346]: https://docs.google.com/document/d/1_DRDzHjmQ5r4d_MWaE4KhTTFwD0cN3OS7IIpgKsadfE/edit?usp=sharing, original [关于052C舰载相控阵雷达背后的故事][https://web.archive.org/web/20240625010706/https://top81.ws/fshow.php?f=1&t=1678204&m=13771040&p=1](关于052C舰载相控阵雷达背后的故事)

**Comment by Luigi-II** [comment]
My friend, I have to say, your ability to gather information is amazing. However, as the DB manager wishes to say, if some data is too close to reality, it may become "less realistic" because of some "limitations" of the simulator. Therefore, we should in some respects rely on the “existing database entries” that can be used as a template or reference to provide equipment’s specifications. Our DB team know what is "reasonable".

**Comment by Luigi-II** [comment]
The game's DB can be said to be "irrelevant" to reality. The vast majority of its data, I think, serves only to achieve a certain "desired effect." While not, you know, absolutely realistic.

**Comment by PygmalionOfCyprus** [comment]
I wouldn't say that...in relation to my comment on #4291 it's moreso just that things get a little wonky when AESA T/R modules get involved, and when we start guestimating power/gain/loss. A little gain/loss goes a long way. To use that issue as an example, giving the radar a peak power of 1192000W and then upping its gain to ~55dB creates a "true" detection range of like...5000+ nmi for a 2m^2 sized target. Obviously this is silly if we've determined that the max instrumented range is ~375nm.

Our DB radar model is actually quite good (at least according to the pros that use it) but it's a very delicate model, which is to say the pieces have to work together as they do IRL and when you start estimating it gets broken very quickly if you're not careful. If you give me a specific source that says, explicitly, the Type 346B has a power of 1192000W and a gain of 55dB I'll plug it in and trust the sim. If we're just guessing at things based on what we think we know ("Well, Radar X has a gain of 10, and this has 20x the number of modules, etc...") and I see numbers like that I'm gonna raise my eyebrows and assume we've guessed wrong somewhere. It might not even be the gain -- it might be some other component of the radar -- but I'm not gonna blindly push the dial up so far so that the results in-sim start to contradict what we DO know.

TL;DR: Radars are complicated, if you don't have all the pieces they're probably gonna break, so I massage guesstimated numbers where necessary to get truer-to-life sim vs. blindly plugging in a suggested number. If you actually have true, proven data I will plug it in without adjusting it so the sim can be the arbiter of truth.

**Comment by RickDNiu** [comment]
> I wouldn't say that...in relation to my comment on #4291 it's moreso just that things get a little wonky when AESA T/R modules get involved, and when we start guestimating power/gain/loss. A little gain/loss goes a long way. To use that issue as an example, giving the radar a peak power of 1192000W and then upping its gain to ~55dB creates a "true" detection range of like...5000+ nmi for a 2m^2 sized target. Obviously this is silly if we've determined that the max instrumented range is ~375nm.
> 
> Our DB radar model is actually quite good (at least according to the pros that use it) but it's a very delicate model, which is to say the pieces have to work together as they do IRL and when you start estimating it gets broken very quickly if you're not careful. If you give me a specific source that says, explicitly, the Type 346B has a power of 1192000W and a gain of 55dB I'll plug it in and trust the sim. If we're just guessing at things based on what we think we know ("Well, Radar X has a gain of 10, and this has 20x the number of modules, etc...") and I see numbers like that I'm gonna raise my eyebrows and assume we've guessed wrong somewhere. It might not even be the gain -- it might be some other component of the radar -- but I'm not gonna blindly push the dial up so far so that the results in-sim start to contradict what we DO know.
> 
> TL;DR: Radars are complicated, if you don't have all the pieces they're probably gonna break, so I massage guesstimated numbers where necessary to get truer-to-life sim vs. blindly plugging in a suggested number. If you actually have true, proven data I will plug it in without adjusting it so the sim can be the arbiter of truth.

What I said was 119.2kW, not 1.192MW. 
Refer to the translation at https://docs.google.com/document/d/1_DRDzHjmQ5r4d_MWaE4KhTTFwD0cN3OS7IIpgKsadfE/edit, original (not source, the author allegedly deleted it soon after posting them in 2016) at https://web.archive.org/web/20240625010706/https://top81.ws/fshow.php?f=1&t=1678204&m=13771040&p=1.

**Comment by PygmalionOfCyprus** [comment]
![image](https://github.com/user-attachments/assets/157fa54c-f766-4ee4-a45c-18cde4827610)

I was referring to issue #4291, since my comment on that issue is (I think) what led to Luigi's comment here.

**Comment by PygmalionOfCyprus** [comment]
Is this issue still being worked on BTW or are we clear to try implementing it?

**Comment by RickDNiu** [comment]
I intended to work on the “pre-modern” (prior to 052B/the last 053) but timing didn’t work out… Modern ones are complete.

**Comment by PygmalionOfCyprus** [comment]
If you're happy with the modern ones, could you do me a favor and break this up into a few separate tickets w/ specific action items? Can just do one for the 052C family, one for the 052D, etc. It's quite time-intensive going through the spreadsheet and trying to figure out what is currently accurate, what needs to be changed, etc., especially as there aren't specific sources to follow back. (For example, it says the 2024 MLU on the 052C added new CIWS; I'd love to see the source/picture/etc. so I can independently verify. I believe you but I want to link the source in the DB.) Likewise, any sensor changes you actually need implemented should get broken into individual tickets.

Even if some of the new tickets is just copy-pasting stuff you've written here it will be much, much easier for me to split/delegate work on this rather than having a single ticket that's going to take me several days to dig through, verify, etc.
